python/ray/tests/BUILD

```diff
@@ -192,6 +191,16 @@ py_test_module_list(
   deps = ["//:ray_lib"],
 )
 
+py_test_module_list(
```

Comment:
nit: can you move this to below the `# Flaky tests only on Mac` comment down below.

Reply:
Good catch :)